<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuClass</class>
 <widget class="QDialog" name="MenuClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menu</string>
  </property>
  <widget class="QPushButton" name="pbStartServer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Start Server</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbConnect">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblHeader">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Start server or connect...</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblIp">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>IP:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPort">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Port:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPortDefault">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>(default port: 5000)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leIp">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>451</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePort">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>451</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblStatus">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Awaiting action...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbStart">
   <property name="geometry">
    <rect>
     <x>444</x>
     <y>240</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Game</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
